code/model/modelanimation.cpp

```diff
@@ -321,6 +321,8 @@ namespace animation {
 		if (!submodel.first || !submodel.second) 
 			return;
 
+		submodel.second->flags.set(Model::Submodel_flags::Can_move);
```

Comment:
I recommend adding a logging statement, if the submodel didn't previously have this flag, indicating that the flag has been added.  Ideally, models should not be modified after being read, but this is a valid and appropriate exception.